libraries/resource_providers/api600/c7000/server_certificate_provider.rb

```diff
@@ -18,7 +18,6 @@ def import_certificate
           @item.data['remoteIp'] = @item.data['name']
           x = @item.get_certificate
           @item.data.clear
-          @item.data['type'] = x['type']
```

Comment:
The type field is not mandatory for this POST call. And this is causing failures with API1600 (since the type has changed), so removed this line. Also confirmed that the functionality is not broken with this change. Tested with API 600 also.